Dockerfile

```diff
@@ -1,14 +1,22 @@
-ARG IMAGE_BUILD_GO=golang:1.20-bullseye
-ARG IMAGE_BASE=gcr.io/distroless/static-debian11
+ARG IMAGE_BUILD_GO=google-go.pkg.dev/golang:1.20.14@sha256:6f86d8a81ff191bee8d3ff8b4c193889560b4ca15df373d5084953c5c860190f
+ARG IMAGE_BASE=gke.gcr.io/gke-distroless/libc@sha256:4f834e207f2721977094aeec4c9daee7032c5daec2083c0be97760f4306e4f88
 
 FROM ${IMAGE_BUILD_GO} AS gobase
 WORKDIR /app
 COPY . ./
-RUN make build
+RUN mkdir /etc/alertmanager
+RUN mkdir /alertmanager
+RUN CGO_ENABLED=1 go build \
+    -tags netgo,boring \
```

Comment:
If we're using a `libc` base, we shouldn't need `netgo`, right?

Reply:
I honestly don't know what the repercussions are of changing it. Do you?

Reply:
Context: https://go.dev/doc/go1.5#net

Looks like this hasn't been necessary for a while, actually.

Reply:
SGTM - done